tutorials/roofit/rf711_lagrangianmorph.C

```diff
@@ -0,0 +1,173 @@
+                              DataError(RooAbsData::None), XErrorSize(0));
+   morph_datahist_0p25.plotOn(frame1, Name("morph_dh_cHq3=0.25"), DrawOption("C"), LineColor(kGreen + 1),
+                              DataError(RooAbsData::None), XErrorSize(0));
+   morph_datahist_0p5.plotOn(frame1, Name("morph_dh_cHq3=0.5"), DrawOption("C"), LineColor(kGreen + 2),
+                             DataError(RooAbsData::None), XErrorSize(0));
+
+   // C r e a t e  w r a p p e d  p d f  t o g e n e r a t e
+   // 2D  d a t a s e t  o f  c H q 3  a s  a  f u n c t i o n  o f
+   // o b s e r v a b l e
+   // ---------------------------------------------
+
+   RooWrapperPdf model("wrap_pdf", "wrap_pdf", morphfunc);
+   RooDataSet *data = model.generate(RooArgSet(cHq3, obsvar), 1000000);
+   TH1 *hh_data = data->createHistogram("pTV vs cHq3", obsvar, Binning(20), YVar(cHq3, Binning(50)));
+   hh_data->SetTitle("Morphing prediction");
+
+   // D r a w  p l o t s  o n  c a n v a s
+   // ---------------------------------------------
+   TCanvas *c1 = new TCanvas("fig3", "fig3", 1200, 400);
+   c1->Divide(3, 1);
+
+   c1->cd(1);
+   gPad->SetLeftMargin(0.15);
+   gPad->SetRightMargin(0.05);
+
+   frame0->Draw();
+   frame0->GetXaxis()->SetTitle("c_{Hq^{(3)}}");
+   TLegend *leg1 = new TLegend(0.55, 0.65, 0.94, 0.87);
+   leg1->SetTextSize(0.04);
+   leg1->SetFillColor(kWhite);
+   leg1->SetLineColor(kWhite);
+   leg1->AddEntry(frame0->findObject("SM_NPsq0"), "SM", "LP");
+   leg1->AddEntry((TObject *)0, "", "");
+   leg1->AddEntry(frame0->findObject("cHq3_NPsq1"), "c_{Hq^(3)}=1.0 at #Lambda^{-2}", "LP");
+   leg1->AddEntry((TObject *)0, "", "");
+   leg1->AddEntry(frame0->findObject("cHq3_NPsq2"), "c_{Hq^(3)}=1.0 at #Lambda^{-4}", "LP");
+   leg1->Draw();
+
+   c1->cd(2);
+   gPad->SetLeftMargin(0.15);
+   gPad->SetRightMargin(0.05);
+
+   frame1->Draw();
+   frame1->GetXaxis()->SetTitle("p_{T}^{V}");
+
+   TLegend *leg2 = new TLegend(0.60, 0.65, 0.94, 0.87);
+   leg2->SetTextSize(0.04);
+   leg2->SetFillColor(kWhite);
+   leg2->SetLineColor(kWhite);
+   leg2->AddEntry(frame1->findObject("morph_dh_cHq3=0.01"), "c_{Hq^{(3)}}=0.01", "L");
+   leg2->AddEntry((TObject *)0, "", "");
+   leg2->AddEntry(frame1->findObject("morph_dh_cHq3=0.25"), "c_{Hq^{(3)}}=0.25", "L");
+   leg2->AddEntry((TObject *)0, "", "");
+   leg2->AddEntry(frame1->findObject("morph_dh_cHq3=0.5"), "c_{Hq^{(3)}}=0.5", "L");
+   leg2->AddEntry((TObject *)0, "", "");
+   leg2->Draw();
+
+   c1->cd(3);
+   gPad->SetLeftMargin(0.12);
+   gPad->SetRightMargin(0.18);
+   gStyle->SetNumberContours(255);
+   gStyle->SetPalette(kGreyScale);
+   gStyle->SetOptStat(0);
+   TColor::InvertPalette();
+   gPad->SetLogz();
+   hh_data->GetYaxis()->SetTitle("c_{Hq^{(3)}}");
+   hh_data->GetYaxis()->SetRangeUser(0, 0.5);
+   hh_data->GetXaxis()->SetTitle("p_{T}^{V}");
+   hh_data->GetZaxis()->SetTitleOffset(1.8);
+   hh_data->Draw("COLZ");
+   c1->SaveAs("rf711_lagrangianmorph.pdf");
```

Comment:
```suggestion
   c1->SaveAs("rf711_lagrangianmorph.png");
```
The other tutorials export `*.png`. I know pdf is nicer, but I'd rather stay consistent with the other tutorials. In fact for me as a developer it's more convenient, because I ran remove all plots with `rm *.png` and open then remotely with lightweight image viewers.

Reply:
Actually I forgot to make the same comment in rf710_roopoly.C/.py before. Can you change it in these files too? Can be the same PR.